### DISCUSSION TITLE
Transcription language

### DISCUSSION THREAD

**Discussion by msnnvnr** [maxkrieger/voiceliner#74] [General]
On ios the app doesn't uses the System language to Transcribe recordings as the app description says. System language is set to german but Voiceliner transcribes in english. How can it be changed?

**Comment by maxkrieger** [comment]
I recently rolled out an update which allows transcription language to be set. Can you go to the app's settings (click the ⚙️) and change it there? If not, wait for the update to roll out.

> as the app description says

Where does it say this?

**Reply by msnnvnr** [reply]
Hi Max, 
thanks a lot for implementing this settings.
I read somewhere that it follows System settings. Don't know where.

Could you add an Option to delete outlines?

**Reply by maxkrieger** [reply]
Archive it first!

**Reply by msnnvnr** [reply]
Thanks. Works. However an "delete" option would be also nice.

**Reply by maxkrieger** [reply]
You can delete archived outlines :)
